jobs/JGDAS_ATMOS_CHGRES_FORENKF

```diff
@@ -17,31 +17,19 @@ export DO_CALC_ANALYSIS=${DO_CALC_ANALYSIS:-"YES"}
 # Begin JOB SPECIFIC work
 ##############################################
 
-GDATE=$(${NDATE} -${assim_freq} ${CDATE})
-gPDY=$(echo ${GDATE} | cut -c1-8)
-gcyc=$(echo ${GDATE} | cut -c9-10)
-GDUMP=${GDUMP:-"gdas"}
-
-export OPREFIX="${CDUMP}.t${cyc}z."
-export GPREFIX="${GDUMP}.t${gcyc}z."
-export APREFIX="${CDUMP}.t${cyc}z."
+export APREFIX="${CDUMP/enkf}.t${cyc}z."
+export APREFIX_ENS="${CDUMP}.t${cyc}z."
 
 if [ ${RUN_ENVIR} = "nco" -o ${ROTDIR_DUMP:-NO} = "YES" ]; then
-    export COMIN=${COMIN:-${ROTDIR}/${RUN}.${PDY}/${cyc}/atmos}
-    export COMOUT=${COMOUT:-${ROTDIR}/${RUN}.${PDY}/${cyc}/atmos}
-    export COMOUT_ENS=${COMOUT_ENS:-${ROTDIR}/enkfgdas.${PDY}/${cyc}}
-    export COMIN_OBS=${COMIN_OBS:-$(compath.py ${envir}/obsproc/${obsproc_ver})/${RUN}.${PDY}/${cyc}/atmos}
-    export COMIN_GES_OBS=${COMIN_GES_OBS:-$(compath.py ${envir}/obsproc/${obsproc_ver})/${GDUMP}.${gPDY}/${gcyc}/atmos}
+    export COMOUT=${COMOUT:-${ROTDIR}/${RUN/enkf}.${PDY}/${cyc}/atmos}
+    export COMOUT_ENS=${COMOUT_ENS:-${ROTDIR}/${RUN}.${PDY}/${cyc}}
 else
-    export COMOUT="${ROTDIR}/${CDUMP}.${PDY}/${cyc}/atmos"
-    export COMOUT_ENS="${ROTDIR}/enkfgdas.${PDY}/${cyc}"
-    export COMIN_OBS="${COMIN_OBS:-${ROTDIR}/${CDUMP}.${PDY}/${cyc}/obs}"
-    export COMIN_GES_OBS="${ROTDIR}/${GDUMP}.${gPDY}/${gcyc}/obs"
+    export COMOUT="${ROTDIR}/${RUN/enkf}.${PDY}/${cyc}/atmos"
+    export COMOUT_ENS="${ROTDIR}/${RUN}.${PDY}/${cyc}"
 fi
 mkdir -m 775 -p ${COMOUT}
-# COMIN_GES and COMIN_GES_ENS are used in script
-export COMIN_GES="${ROTDIR}/${GDUMP}.${gPDY}/${gcyc}/atmos"
-export COMIN_GES_ENS="${ROTDIR}/enkfgdas.${gPDY}/${gcyc}"
+
+export ATMFCST_ENSRES="${COMOUT_ENS}/mem001/atmos/${APREFIX_ENS}atmf006.nc"
```

Comment:
I believe it is just needs an example of the file so it can extract the grid information. Since the number of members can vary, it is best to look for the first member, since that will be there no matter what the ensemble size.

Reply:
Ok, gotcha, thanks!